cmd/farva-monitor/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/bcwaldon/farva/pkg/monitor"
+)
+
+func main() {
+	fs := flag.NewFlagSet("farva-monitor", flag.ExitOnError)
+
+	cfg := monitor.DefaultConfig
+
+	fs.DurationVar(&cfg.RefreshInterval, "refresh-interval", cfg.RefreshInterval, "Attempt to sync AWS entities at this interval")
```

Comment:
This was a bit confusing to me: we're both assigning to cfg.RefreshInterval and using it as a default value. It's
equivalent to `a = input or a` so maybe it's idiomatic but as someone new to go it's kind of confusing. I think just using `monitor.DefaultConfig.RefreshInterval` would be more readable.